### ISSUE TITLE
Incorrect unit not detected

### ISSUE THREAD

**Issue by Cuadratin** [IDPF/epubcheck#529]
When validating epubcheck ignores an incorrect CSS syntax. 

I've got "1m" instead of "1em" and the epub validated with no problems or warnings (it crashed when opening with Adobe Digital Editions).

**Comment by tofi86** [comment]
This is falsely reported as

```
USAGE(ACC-014): ./src/test/resources/minimal-epub/30/minimal-epub-30.epub/OPS/stylesheet.css(5,2):
Value of CSS property 'font-size' does not use a relative size.
```
in [EpubCSSCheckCSSHandler.java#L566](https://github.com/IDPF/epubcheck/blob/master/src/main/java/com/adobe/epubcheck/ctc/css/EpubCSSCheckCSSHandler.java#L566)

which is kind of true as EpubCheck recognizes `1m` as `DIMEN` (_dimension_) which technically is a valid property value but in this case an invalid e.g. font-size property.

**Comment by tofi86** [comment]
Fixed this with PR #776

**Comment by tofi86** [comment]
Fixed with PR #776
